torch/optim/lr_scheduler.py

```diff
@@ -314,6 +316,26 @@ def get_lr(self):
             return list(self.base_lrs)
 
 
+class WarmUpLR(LambdaLR):
```

Comment:
Do you have an example code where you would use this? This looks very close to simply using `MultiplicativeLR` with those formula. Thoughts?

Reply:
``` python
warmup = WarmUpLR()
for epoch in range(50):
  train(...)
  val(...)
  if epoch < 5:
    warmup.step()
```
In the first 5 epoch, I would use warmup to gradually increase learning rate from low to its origin value. It is normally used in vision task to stable the training process. It would be nice to have such a API. Also, I don't think it is chainable.